packages/monaco/src/browser/monaco-loader.ts

```diff
@@ -11,17 +11,19 @@ export function loadVsRequire(context: any): Promise<any> {
     const originalRequire = context.require;
 
     return new Promise<any>(resolve => {
-        const vsLoader = document.createElement('script');
-        vsLoader.type = 'text/javascript';
-        vsLoader.src = './vs/loader.js';
-        vsLoader.charset = 'utf-8';
-        vsLoader.addEventListener('load', () => {
-            // Save Monaco's amd require and restore the original require
-            const amdRequire = context.require;
-            context.require = originalRequire;
-            resolve(amdRequire);
-        });
-        document.body.appendChild(vsLoader);
+        window.onload = () => {
```

Comment:
hello,akosyakov,I want to kown why we add onLoad event listener here?I had removed load event listener here, but nothing goes wrong;

Reply:
to ensure that `document` is loaded, otherwise `document.body.appendChild` won't work, we can remove it if we move bundle.js into body

Reply:
Thanks 😊